docs/summer-of-code-ideas/summer-of-code-reports/gsoc-2020-andreperezmaselco.md

```diff
@@ -0,0 +1,203 @@
+# Google Summer of Code 2020 Report
+
+##### Project:
+##### [Fuzzing Graphics Shaders](https://summerofcode.withgoogle.com/projects/#6594591471435776)
+
+##### Student:
+##### [André Perez Maselco](https://github.com/andreperezmaselco)
+
+##### Organization:
+##### [Android Graphics Tools Team](https://summerofcode.withgoogle.com/organizations/6102438181863424/)
+
+##### Mentors:
+##### [Alastair Donaldson](https://github.com/afd)
+##### [Paul Thomson](https://github.com/paulthomson)
+
+## Abstract
+
+During the *Google Summer of Code 2020* program, I worked on the development of new semantics-preserving transformations, fuzzer passes and enhancements for the *spirv-fuzz* tool.
```

Comment:
Perhaps you could link to this URL for spirv-fuzz:

https://github.com/google/graphicsfuzz/blob/master/docs/finding-a-vulkan-driver-bug-using-spirv-fuzz.md

Reply:
Done.